sdk/python/kubeflow/storage_initializer/hugging_face.py

```diff
@@ -77,11 +94,19 @@ def load_config(self, serialised_args):
         self.config = HfDatasetParams(**json.loads(serialised_args))
 
     def download_dataset(self):
-        print("downloading dataset")
+        logger.info("Downloading dataset")
+        logger.info("-" * 40)
         import huggingface_hub
         from datasets import load_dataset
 
         if self.config.access_token:
             huggingface_hub.login(self.config.access_token)
 
         load_dataset(self.config.repo_id, cache_dir=VOLUME_PATH_DATASET)
```

Comment:
@andreyvelich why are we downloading the dataset again

Reply:
It's great catch @deepanker13!
We should remove it.